suite-native/app/e2e/README.md

```diff
@@ -0,0 +1,53 @@
+
+Detox configuration can be found in the [.detoxrc.js](../.detoxrc.js) file. `jest` is used as the test runner.
+
+The tests Are always executed with the env variable `IS_DETOX_BUILD` equal to `"true"`. So in case you would need to determine if the JS code is running within a test build, you can always access this variable. For example, it is used in a debug build to hide react-native LogBox, so it is not blocking the UI for the test runner.
+
+## Running tests locally
+
+To run the tests locally, you need to have the following installed:
+
+-   for **iOS** build: Installed Xcode with iPhone 11 simulator created
+-   for **Android** build: Installed Android Studio with existing emulator device named `Pixel_3a_API_31`. You can do it via Android Studio UI. If you want to do it via the command line, please follow [this guide](https://wix.github.io/Detox/docs/guide/android-dev-env#heres-how-to-install-them-using-the-command-line). You can also use another emulator device, but you have to map its name to the `devices.emulator.device.avdName` value in the `.detoxrc.js` config file.
+
+### Debug build
+
+With the debug config, the app is running in Expo dev-client and the JavaScript bundle is served from the local Metro server. This configuration is suitable for development and testing purposes because the Metro hot reloads make it easy to debug the test scenarios. To run the tests in debug mode, follow these steps:
+
+1. Navigate to the `suite-native/app` folder
+2. generate Expo native code with `yarn prebuild:clean`
+3. Create a debug build:
+    - (**Android**) `yarn build:e2e android.emu.debug`
+    - (**iOS**) `yarn build:e2e ios.sim.debug`
+4. Run Metro server: `yarn start`
+5. While is Metro server running in a different console, execute the E2E tests:
+    - (**Android**) `yarn test:e2e android.emu.debug`
+    - (**iOS**) `yarn test:e2e ios.sim.debug`
+
+### Release build
+
+To test the app in the release mode, you need to build the app with the release configuration. The JavaScript bundle is bundled within the app and the app is running in the standalone mode. To run the tests in the release mode, follow these steps:
+
+1. Navigate to the `suite-native/app` folder
+2. generate Expo native code with `yarn prebuild:clean`
+3. Create a release build:
+    - (**Android**) `yarn build:e2e android.emu.release`
+    - (**iOS**) `yarn build:e2e ios.sim.release`
+4. Execute the E2E tests:
+    - (**Android**) `yarn test:e2e android.emu.release`
+    - (**iOS**) `yarn test:e2e ios.sim.release`
+
+> BEWARE: Sometimes the first execution of debug build fail, because the bundling on the metro server takes longer for the first time. If you encounter this behavior, just rerun the tests and the second run should go smoothly.
```

Comment:
`> BEWARE: Sometimes the first execution of debug build fail` do you mean `debug` or `release`? 

I guess you meant release since it's under release build, but is this first-time fail because of the jest config test timeout? If it's known problem and could be solved by bumping the timeout, I'd do it.

Reply:
It was ment to be in the `debug` section my bad. It is related to Metro timeout, so extending Jest timeout would have no effect. Fixed in 5d9543b375

Reply:
Ok, cool. For debug its fine if first run fails occasionaly.